### PR TITLE
add deploymon pipeline plugin repo

### PR DESCRIPTION
**Pull request by hoegertn**
# Description

Permission PR for https://issues.jenkins-ci.org/browse/HOSTING-477
Repo: https://github.com/jenkinsci/pipeline-deploymon-plugin

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@hoegertn have you completed the checklist suggested in the PR template?

**Comment by hoegertn** [comment]
Yes and all information should be in the description.
The Wiki page for the plugin will be published tomorrow and I will release the first version shortly after.

**Comment by oleg-nenashev** [comment]
Added a too-late comment to the hosting request: Plugin lists source: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/jenkins/install/platform-plugins.json .

File: permissions/plugin-pipeline-deploymon.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "pipeline-deploymon"
+paths:
+- "de/taimos/pipeline-deploymon"
```
Comment (oleg-nenashev) [inline permissions/plugin-pipeline-deploymon.yaml]: Would be nice to have "jenkins/plugins" or so in the Group ID.

**Comment by hoegertn** [comment]
@oleg-nenashev do I have to do something or is this merged in the near future?

**Comment by oleg-nenashev** [comment]
@hoegertn no, it is fine. The groupId comment is up to you.

**Comment by hoegertn** [comment]
I still get Access Denied when trying to do a `mvn deploy`. Settings are correct because my other plugin works.

**Comment by oleg-nenashev** [comment]
@hoegertn make sure you have logged in to https://repo.jenkins-ci.org/webapp/#/home. It was described in the checklist. If you have logged in, check the credentials configuration in you Maven settings.xml

**Comment by hoegertn** [comment]
Yes I logged in but it still says `Access denied`. I will try again later.

**Comment by oleg-nenashev** [comment]
Make sure you have credentials set in settings.xml:

e.g.
```
<server>
      <id>maven.jenkins-ci.org</id>
      <username>hoegertn</username>
      <password>{my_encoded_password=}</password>
    </server>
```
Where password is your encoded or plaintext Jenkins JIRA password.

Just in case, what do you run: `mvn clean deploy` or `mcn release:prepare release:perform`? Maybe "deploy" in your comment confused me. If you are running release, somebody from @jenkins-infra/contributors needs to check the trusted CI logs. I have no access to them. Maybe the permission update logic is disabled/broken since the Artifactory outages last week. CC @rtyler @olblak

**Comment by hoegertn** [comment]
I tried `mvn clean deploy -P jenkins_release` in the tag and `mvn release:prepare release:perform`.

Settings are set and a release of the `pipeline-aws` plugin worked last week. So IMHO it is a permission thing.

**Comment by daniel-beck** [comment]
The file extension must be `.yml`, cannot be `.yaml`.

The checklist exists for a reason.

**Comment by daniel-beck** [comment]
I renamed the file. Expect permissions to be applied within ~30 minutes.

**Comment by hoegertn** [comment]
Thanks. Seems like my IDE renamed the extension. I originally copied another file as reference.

**Pull request merged**